activiti-cloud-services-runtime-bundle/activiti-cloud-services-subscriptions/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.activiti.spring.bpmn.parser.CloudActivityBehaviorFactoryInterceptor =\
```

Comment:
@daisuke-yoshimoto again, this is all good things but for now let's stick with the current mechanism, so we can refactor after beta1 release.